tensorflow-core.yaml

```diff
@@ -144,7 +144,11 @@ subpackages:
           _pyver=$(python -c 'import sys; print(str(sys.version_info[0]) + "." + str(sys.version_info[1]))')
           _pypath="/usr/lib/python${_pyver}"
           mkdir -p "${{targets.contextdir}}"/usr/lib
+          mkdir -p "${{targets.contextdir}}"/usr/bin
           mv "${{targets.destdir}}"${_pypath} "${{targets.contextdir}}"${_pypath}
+          mv "${{targets.destdir}}"/usr/bin "${{targets.contextdir}}"/usr/bin
+          find "${{targets.contextdir}}"${_pypath} -name "*.pyc" -exec rm -rf '{}' +
```

Comment:
We switched in python base packages to _not_ remove these files. lack of .pyc files means that every python startup includes the cost of creating them (even if there is no place to write them).

Reply:
rollbacked

Reply:
@smoser we need to update [some](https://github.com/search?q=repo%3Awolfi-dev%2Fos+*.pyc&type=code) melange definitions.